adding note to cover COPS-6883 being resolved (#3510)
mesosphere/dcos-docs-site: pages/dkp/konvoy/1.7/release-notes/1.7.1/index.md

A problem with the `konvoy down` or `konvoy reset` commands failing if dedicated named devices were mounted on `/var/lib/kublet` or `/var/lib/containerd` has been corrected. (COPS-6771)

#### Fix issue reporting incorrect status update of addons upon deploying

> There was an issue when Addons were marked with a status of "OK" when in certain cases not all components of that addon were available during the `Deploying Enabled Addons` stage of a `konvoy up` or `konvoy deploy addons` command. This has been fixed and when the status now reflects when the addon is ready. (COPS-6883)

Reviewer: ```suggestion
There was an issue where some Addons were marked as "OK" status when not all components of that addon were available. This happened during the `Deploying Enabled Addons` stage of a `konvoy up` or `konvoy deploy addons` command. This has been fixed and the status now reflects when the addon is ready. (COPS-6883)
```
